src/config/configuration.ts

```diff
@@ -1,11 +1,15 @@
-import 'dotenv/config';
+const get = (key: string) => {
+  return process.env[key];
+};
+
 export default () => ({
   database: {
-    host: process.env.DATABASE_HOST || '',
-    user: process.env.DATABASE_USER || '',
-    name: process.env.DATABASE_NAME || '',
-    password: process.env.DATABASE_PASSWORD || '',
-    port: process.env.DATABASE_PORT || '',
-    schema: process.env.DATABASE_SCHEMA || '',
+    host: get('DATABASE_HOST') ?? '',
+    user: get('DATABASE_USER') ?? '',
+    name: get('DATABASE_NAME') ?? '',
+    password: get('DATABASE_PASSWORD') ?? '',
+    port: get('DATABASE_PORT') ?? '',
+    schema: get('DATABASE_SCHEMA') ?? '',
   },
+  port: +get('PORT') ?? 3000,
```

Comment:
`PORT` 값이 없을 시 `+get('PORT')` 의 값이 NaN 이 되어버려 기본값이 설정이 안되네요..! `+` 기호를 삭제하는 게 좋을 것 같습니다.

```suggestion
  port: get('PORT') ?? 3000,
```

Reply:
이렇게 하면 포트가 number이거나 string이어서 문제가 될 것 같았는데 `app.listen()` 함수가 포트를 두 가지 타입으로 받네요. 문제 없을 것 같습니다.